io.github.bit_team.back_in_time.gui.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Docu:
+        https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html
+
+    Validate (lint) on shell:
+        appstreamcli validate *xml
+-->
+
+
+<component type="desktop-application">
+  <id>io.github.bit_team.back_in_time.gui</id>
+  <name>Back In Time</name>
+  <summary>Backup tool for GNU Linux desktop using rsync with its hard-links feature to save storage space</summary>
+
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>GPL-2.0</project_license>
+
+  <description>
+    <p>
+      It is an easy-to-use backup tool for files and folders. It runs on GNU
+      Linux and provides a command line tool "backintime" and a Qt5 GUI
+      "backintime-qt". It uses "rsync" in the back to take manual or scheduled
+      snapshots and stores them locally or remotely through SSH. Each snapshot
+      is its own folder with copies of the original files, but unchanged files
+      are hard-linked between snapshots to save space. It was inspired by
+      "FlyBack".
+    </p>
+    <p>
+      You only need to specify 3 things:
+    </p>
+    <ol>
+      <li>What folders to back up.</li>
+      <li>Where to save snapshots.</li>
+      <li>The backup frequency (manual, every hour, every day, every month).</li>
+    </ol>
+  </description>
+
+    <icon type="stock">document-save</icon>
+
+    <categories>
+      <!-- https://specifications.freedesktop.org/menu-spec/latest/apa.html -->
+      <category>System</category>
+      <category>Archiving</category>
+    </categories>
+
+    <keywords>
+      <keyword translate="no">Backup</keyword>
+      <keyword translate="no">Rsync</keyword>
+      <keyword translate="no">GUI</keyword>
+    </keywords>
+
+    <url type="homepage">https://github.com/bit-team/backintime</url>
+    <url type="bugtracker">https://github.com/bit-team/backintime/issues</url>
+    <url type="faq">https://github.com/bit-team/backintime/blob/dev/FAQ.md</url>
+    <url type="help">https://backintime.readthedocs.io/</url>
+    <url type="translate">https://translate.codeberg.org/engage/backintime</url>
+    <url type="contact">https://mail.python.org/mailman3/lists/bit-dev.python.org</url>
+    <url type="vcs-browser">https://github.com/bit-team/backintime</url>
+    <url type="contribute">https://github.com/bit-team/backintime/blob/dev/CONTRIBUTING.md</url>
+
+    <launchable type="desktop-id">backintime-qt.desktop</launchable>
+
+    <!--
+        - 16:9 ratio
+        - PNG format
+        - Default desktop theme
+        - Include windows only (incl. boders) and use alpha mask for
+          rounded window corners
+        - Without mouse pointer
+        - Add caption (without fullstop)
+        -->
+    <screenshots>
+      <screenshot type="default">
+        <caption>Main window</caption>
+        <image>https://translate.codeberg.org/media/screenshots/bit_16to9_960x540_mainwindow.png</image>
```

Comment:
It is possible to point to screenshots in the local file system.
But I don't like the idea to "install" png files for each users. Waste of resources because of of our users do not use software centers and won't never see that pictures.

It is also possible to use a screenshot folder in our upstream repo or pointing to screenshots used in our documentation at readthedocs.org.

Not sure about a final and secure solution but currently I would keep it that way.